File: src/main/java/world/ultravanilla/commands/DisabledCommand.java
package world.ultravanilla.commands;

import java.util.Collections;
import java.util.List;

public class DisabledCommand extends UltraCommand {

    public DisabledCommand(String name, String permission) {
        super(name, permission);
    }

    @Override
    protected boolean onCommand(String[] args) {
        sendError("Command disabled.");
        return true;
    }

    @Override
    protected List<String> onTabComplete(String[] args) {
        return Collections.emptyList();
    }

    @Override
    public String getUsage(String arg) {
        return "";
    }
}

File: src/main/java/world/ultravanilla/commands/DisabledCommand.scala
package world.ultravanilla.commands

import net.md_5.bungee.api.ChatColor
import world.ultravanilla.UltraVanilla

import java.util.Collections
import java.util

class DisabledCommand(val instance: UltraVanilla, val commandName: String, val permissionNode: String) extends UltraCommand(instance, commandName, permissionNode) {
    errorColor = ChatColor.of("#ff0000")
    override protected def onCommand(args: Array[String]): Boolean = {
        sendError("Command disabled.")
        true
    }
    override protected def onTabComplete(args: Array[String]): util.List[String] = Collections.emptyList
    override def getUsage(arg: String) = ""
}
